src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/EnumsShouldHaveZeroValue.cs

```diff
@@ -164,9 +164,9 @@ private static void CheckNonFlags(INamedTypeSymbol namedType, ImmutableArray<IFi
         internal static IEnumerable<IFieldSymbol> GetZeroValuedFields(INamedTypeSymbol enumType)
         {
             SpecialType specialType = enumType.EnumUnderlyingType.SpecialType;
-            foreach (IFieldSymbol field in enumType.GetMembers().Where(m => m.Kind == SymbolKind.Field))
+            foreach (var m in enumType.GetMembers())
```

Comment:
Same here, just use `.OfType<IFieldSymbol>()` extension

Reply:
Maybe Roslyn is missing Roslyn is missing `INamespaceOrTypeSymbol.GetMember<TMemberType>()`.